src/App.jsx

```diff
@@ -55,8 +57,8 @@ function App() {
         
         <ReflexElement flex={0.55}>
           <Item style={{width: "100%", height:"100%"}}>
-            {!llmMode && <BlocklyInterface/>}
-            {llmMode && <GPTPanel />}
+            {fullScreenPanel ? <GPTPanel /> : null}
+            <BlocklyInterface />
```

Comment:
We'll want to make sure the BlocklyInterface isn't rendered if the fullScreenPanel is true. Just do:
```suggestion
            {!fullScreenPanel && <BlocklyInterface />}
```